packages/cockpit/src/transactions/metrics/result.js

```diff
@@ -58,34 +60,41 @@ const getNamedWeekdays = map(evolve({
   key: weekdayNumber => getLocalizedWeekdayName(moment().day(weekdayNumber)),
 }))
 
-const buildDefaultWeekdays = map(
-  applySpec({
-    key: identity,
-    doc_count: always(0),
-  }),
+const buildDefaultWeekdays = reduce(
+  (acc, weekdayNumber) => mergeRight({ [weekdayNumber]: 0 }, acc),
+  {},
   range(0, 7)
 )
 
+const aggregateWeekdaysVolume = reduce((acc, {
+  doc_count: volume,
+  key,
+}) => mergeRight(acc, {
```

Comment:
Isso aqui poderia ser mais simples, pois como está sendo forçado o objeto da direita com o `mergeRight`.
```suggestion
reduce((acc, {
  doc_count: volume,
  key,
}) => ({
  [key]: propOr(0, key, acc) + volume
})
```

Reply:
Boa, corrigido!